src/plugins/streaming-plugin.js

```diff
@@ -691,6 +712,11 @@ class StreamingHandle extends Handle {
  * @property {number} [video_port_3] - The port for RTP video (simulcast)
  * @property {number} [video_rtcp_port] - The port for RTCP video
  * @property {number} [data_port] - The port for datachannels
+ * @property {object[]} [ports] - [multistream] The ports used by the RTP mountpoint
```

Comment:
Please fix according to [the API](https://github.com/meetecho/janus-gateway/blob/db86cf7c12b3e30d84b6ead42e2fc128c1b516d3/src/plugins/janus_streaming.c#L4380).

Mandatory:
- type
- mid

Optional:
- msid
- host
- port
- rtcp_port
- port_2
- port_3

If you are not sure about the meaning of some attributes, leave a blank description.